### ISSUE TITLE
Why do the same object classification scores differ greatly？

### ISSUE THREAD

**Issue by hh123445** [open-mmlab/mmrotate#455]
![image](https://user-images.githubusercontent.com/90760124/183424023-75dc648e-3e2f-4846-943f-f082ac184815.png)
A  Insulator score:0.68
B Inuslator score:0.02

![漏检图片](https://user-images.githubusercontent.com/90760124/183426678-69faa132-4070-481d-a18d-16d37c655b40.png)
A  Insulator score:0.6
B Inuslator score:0.02

the result is set score_thr = 0.

I am using rotate FOCS to detect insulators, but I have encountered a problem. As shown in the figure, for insulator strings with similar shape and angle, their classification scores are very different, resulting in missing inspection of insulator strings with low scores. In fact, their positioning is very accurate.

**Comment by zytx121** [comment]
@hh123445 It's a good question! Community discussion is welcome!

**Comment by hh123445** [comment]
Yes, I have tried other detectors, but I still get similar results. I am still very confused about this problem. What do you mean by rotation enhancement? Is it a parameter setting in the program?

**Comment by hh123445** [comment]
> @hh123445这是个好问题！你试过其他探测器吗？也许您可以尝试在数据集上应用旋转增强。

Yes, I have tried other detectors, but I still get similar results. I am still very confused about this problem. What do you mean by rotation enhancement? Is it a parameter setting in the program?

**Comment by zytx121** [comment]
You can refer to this [config](https://github.com/open-mmlab/mmrotate/blob/main/configs/roi_trans/roi_trans_r50_fpn_1x_dota_ms_rr_le90.py) to add `PolyRandomRotate` in your train_pipeline.
